crm_phonecall_planner/wizards/crm_phonecall_planner.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from __future__ import division
+from datetime import datetime, timedelta
+from logging import getLogger
+from openerp import _, api, fields, models
+from openerp.exceptions import ValidationError
+from openerp.tools import safe_eval
+
+_logger = getLogger(__name__)
+
+
+class CrmPhonecallPlan(models.TransientModel):
+    _name = "crm.phonecall.planner"
+    _description = "Phonecall planner"
+    _inherit = [
+        "utm.mixin",
+    ]
+
+    name = fields.Char(
+        "Call Summary",
+        required=True,
+    )
+    user_id = fields.Many2one(
+        comodel_name="res.users",
+        string="Responsible",
+    )
+    team_id = fields.Many2one(
+        comodel_name="crm.team",
+        string="Sales Team",
+    )
+    tag_ids = fields.Many2many(
+        comodel_name="crm.lead.tag",
+        string="Tags",
+    )
+    res_partner_domain = fields.Char(
+        "Partners filter",
+        help="Filter the parters that will get a scheduled call.",
+    )
+    duration = fields.Float(
+        string="Call duration",
+        default=lambda self: self._default_duration(),
```

Comment:
Don't do this, but move to up the default method and just put `default=_default_duration` (without quotes in the method).

Reply:
Nope, check the guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#fields

(same in comments below)

Reply:
I think this guideline is deprecated. Have you tried to inherit now in v9/v10 directly the method to see if it works?

Reply:
Well, I remember to have tried in the past with a core model that defined a lambda, and I had to redefine the field just to override the default method.

The problem when you do `default=_default_duration` is that you bind directly the function code object. By using a lambda, you bind directly the lambda code, but inside it you have the benefits of all the inheritance system. That's the reason behind that guideline.

Other field attributes, such as `compute` or `search`, accept a string that defines the method name and make this wrapper automatically, but in `default` that system is not available, since passing a string would use that string as the default value instead, so the lambda callback wrapper is the best choice here AFAIK.

Reply:
OK, yes, it makes sense